### PR TITLE
Vehicle Data Additions

### PR DESCRIPTION
**Pull request by atiwari9**
This proposal is to add new vehicle data items to SDL.

### CODE REVIEW

**Comment by atiwari9** [comment]
@theresalech  - Ready for review

File: proposals/NNNN-Vehicle-data-additions.md [hunk 1]
```diff
@@ -0,0 +1,1341 @@
+# Vehicle Data Additions
+
+* Proposal: [SDL-NNNN](NNNN-Vehicle-data-additions.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | SDL SERVER | SHAID | PROXY]
```
Comment (theresalech) [inline proposals/NNNN-Vehicle-data-additions.md]: Please change "PROXY" to specific platforms.  For example, Java Suite, iOS, JavaScript Suite.

File: proposals/NNNN-Vehicle-data-additions.md [hunk 2]
```diff
@@ -0,0 +1,1341 @@
...
+
+## Introduction
+
+This proposal is to add new vehicle data items to SDL.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional set of vehicle data item through SDL. This goes in line with commitment to enhance SDL with even more rich vehicle data content.
```
Comment (theresalech) [inline proposals/NNNN-Vehicle-data-additions.md]:
```suggestion
In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even more rich vehicle data content.
```

File: proposals/NNNN-Vehicle-data-additions.md [hunk 3]
```diff
@@ -0,0 +1,1341 @@
...
+
+## Proposed Solution 
+
+We need to add some new vehicle data items. Following are the details on all the changed proposed.
```
Comment (theresalech) [inline proposals/NNNN-Vehicle-data-additions.md]:
```suggestion
We need to add some new vehicle data items. Following are the details on all of the changes proposed.
```

File: proposals/NNNN-Vehicle-data-additions.md [hunk 4]
```diff
@@ -0,0 +1,1341 @@
...
+
+#### New vehicle data items:
+1. `WindowStatus`
+2. `SeatOccupancy`
+3. `StabilityControlsStatus`
+4. `GearStatus`
+5. `ClimateData`
+6. `handsOffSteering`
+7. `hvBatteryLevel`
+8. Additions to `BodyInformation` struct
```
Comment (theresalech) [inline proposals/NNNN-Vehicle-data-additions.md]: Each new vehicle data item should have its own proposal.  Please break this proposal up into separate proposals to ensure each new vehicle data item can be reviewed in detail by SDLC Members, as it’s unreasonable to expect that each OEM can simultaneously review if all of the new vehicle data structures will work with their vehicle data integrations or future integrations.  This is a similar concept to adding new Remote Control modules as individual proposals, as discussed here: https://github.com/smartdevicelink/sdl_evolution/issues/643#issuecomment-447925723.

**Comment by atiwari9** [comment]
I have split this proposal to 8 new ones for:

1. [`StabilityControlsStatus`](https://github.com/smartdevicelink/sdl_evolution/pull/832)
2. [`hvBatteryLevel`](https://github.com/smartdevicelink/sdl_evolution/pull/833)
3. [`BodyInformation` enhancements](https://github.com/smartdevicelink/sdl_evolution/pull/834)
4. [`WindowStatus`](https://github.com/smartdevicelink/sdl_evolution/pull/835)
5. [`SeatOccupancy`](https://github.com/smartdevicelink/sdl_evolution/pull/836)
6. [`GearStatus`](https://github.com/smartdevicelink/sdl_evolution/pull/837)
7. [`ClimateData`](https://github.com/smartdevicelink/sdl_evolution/pull/838)
8. [`handsOffSteering`](https://github.com/smartdevicelink/sdl_evolution/pull/839)

This PR can be closed

**Comment by theresalech** [comment]
Closing in favor of separate proposals entered for each vehicle data addition, linked [here](https://github.com/smartdevicelink/sdl_evolution/pull/830#issuecomment-538060271).

**Pull request closed without merging**